src/markdown/features/data-management.md

```diff
@@ -29,6 +29,25 @@ Delete a document
 - DELETE `${api.url}/app/colls/${coll}/docs/${doc._id}`
 ```
 
+Update many document, give filter and update in body
```

Comment:
This is not part of the CRUD but the advanced Mongo functions (write after that ^^). Find is already documented